test/metabase/upload_test.clj

```diff
@@ -1942,11 +1951,12 @@
                         "1  , 1.0"]
               file     (csv-file-with csv-rows (mt/random-name))]
           (is (some? (replace-csv! {:file file, :table-id (:id table)})))
-          ;; For MySQL the primary key incrementer will be reset, but for Postgres and H2 it will not.
-          (let [pk-offset (case driver/*driver* :mysql -1 0)]
-            (is (= [[(+ pk-offset 2) 1 1]
-                    [(+ pk-offset 3) 1 1]]
-                   (rows-for-table table))))
+          (let [table-rows (rows-for-table table)
+                ;; For MySQL the primary key incrementer will be reset, but for Postgres and H2 it will not.
```

Comment:
I think you can remove this line, especially if you'll be testing it seriously in another driver-specific test